# Request empty for now, these could be useful information to manage
# multi-robot situations but are not relevant right now
# string robot_name
# geometry_msgs/Pose robot_current_pose
---
geometry_msgs/Pose pick_up_location
geometry_msgs/Pose drop_off_location
string guest_name
# std_msgs/ColorRGBA guest_clothes_color
